pages/account/statistics.js

```diff
@@ -75,21 +110,7 @@ export async function getServerSideProps(context) {
     100
   ).toFixed(0);
 
-  data.links.individual = data.links.individual.filter((link) =>
-    profile.links.some((pLink) => pLink.url === link.url)
-  );
-
-  const totalClicks = data.links.individual.reduce((acc, link) => {
-    return acc + link.clicks;
-  }, 0);
-  data.links.clicks = totalClicks;
-
-  data.profile.daily = data.profile.daily.slice(-30).map((day) => {
-    return {
-      views: day.views,
-      date: day.date,
-    };
-  });
+  data.profile.daily = last30DaysStats;
```

Comment:
I have error in DynamicChart component. It gets already formatted date from props and returns invalid date in its formatDate util and the page crashes. Attaching the screenshots
![image](https://github.com/EddieHubCommunity/BioDrop/assets/78021348/2569c9c5-1c7d-4310-a5e3-a3e020291170)
![image](https://github.com/EddieHubCommunity/BioDrop/assets/78021348/abcdf66e-c33d-45a6-88c0-3a25a051c65d)
![image](https://github.com/EddieHubCommunity/BioDrop/assets/78021348/86386171-b9d8-40df-bb93-5d0c9aef43af)

Reply:
In which format do you want the date?

Reply:
I'm not sure I understood you. what do you mean?

Reply:
Sorry, StatsChart

Reply:
"I have added the checks for the date. Can you please review it?

Reply:
I have added the date checks. Please review it.